sdk/client/src/lib.rs

```diff
@@ -40,6 +40,10 @@ cfg_async_or_blocking! {
     pub use move_deserialize::Event;
 }
 
+cfg_websocket! {
+    pub mod stream;
+}
+
```

Comment:
Please mark this similarly to the verifying client since this is still highly experimental and subject to change.